provider/resource_keycloak_openid_client_test.go

```diff
@@ -622,6 +637,21 @@ resource "keycloak_openid_client" "client" {
 	`, realm, clientId)
 }
 
+func testKeycloakOpenidClient_AccessToken_basic(realm, clientId string) string {
+	return fmt.Sprintf(`
+resource "keycloak_realm" "realm" {
+	realm = "%s"
+}
+
+resource "keycloak_openid_client" "client" {
+	client_id   		  = "%s"
+	realm_id    		  = "${keycloak_realm.realm.id}"
+	access_type 		  = "CONFIDENTIAL"
+	access_token_lifespan = "1800"
```

Comment:
Since this test HCL is specifically asserting against the access token lifespan, could you pass this in as an argument?

Reply:
Done :)